### PR TITLE
New Cocktail: The Nar'Sour

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
add: A new cocktail: The Nar'Sour. Made from blood, demonsblood, and lemon juice, this cocktail is known to induce a slightly different type of slurring when imbibed. 
/:cl:

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
i like the sprite

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -547,4 +547,12 @@
...
 	id = "eggnog"
 	results = list("eggnog" = 15)
-	required_reagents = list("rum" = 5, "cream" = 5, "eggyolk" = 5)
\ No newline at end of file
+	required_reagents = list("rum" = 5, "cream" = 5, "eggyolk" = 5)
+
+/datum/chemical_reaction/narsour
+	name = "Nar'sour"
+	id = "narsour"
+	results = list("hearty_punch" = 1)  //Very little, for balance reasons
```
Comment (ACCount12) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: you sure this is right?

Comment (Robustin) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: honk

**Comment by FantasticFwoosh** [comment]
It would not kill you to attach a picture to your PR, looks good though

**Comment by Jordie0608** [comment]
@FantasticFwoosh https://github.com/tgstation/tgstation/pull/34690#issuecomment-359162283

**Comment by Supermichael777** [comment]
I can see some chapo traitor potential, give some to a traitor buddy and get into secs good book. Hack the library and get him a tome or whatever.

**Comment by ExcessiveUseOfCobblestone** [comment]
ARM

**Comment by coiax** [comment]
Remember that REAL (from a talisman) cultslurring also has stammering, if you're going for authenticity.

I mean, I tried to make a bottle of effectively this for 2 TC in #24999, and no one wanted to merge it, so good luck.

**Comment by Repukan** [comment]
Hey let's make the most miserable job in the game even more miserable

**Comment by Repukan** [comment]
Lmao my nigga FUCK sec players

**Comment by Robustin** [comment]
I see a difference between trying to get traitors to use unholy water as a mute toxin and a meme drink for the bartender. Drinks served at the bar are usually done so publicly and the solitary nature of the source (bartender) means its still easy to discern if its legit or not. 

With the loss of colored paper and crayon runes as sources of cult hoax, I felt this was a reasonable addition.

**Comment by Repukan** [comment]
It's not solitary anyone can mix drinks.

We even have abandoned bars coded in every map fucking lmao

**Comment by Repukan** [comment]
This isn't Bay or lifeweb where you can get away with saying only one person in the station is capable of x thing

**Comment by epochayur** [comment]
pretty sure the booze/soda dispenser isn't locked, and the booze fridge can definitely be hacked to not require ID access

**Comment by Robustin** [comment]
You need the Booze-O-Mat to make this, dispenser doesn't get you all the necessary parts.

**Comment by PKPenguin321** [comment]
The only weird thing about this is that it requires blood when demonsblood, another requirement, already requires blood so if you think about it you're mixing blood into blood

**Comment by ExcessiveUseOfCobblestone** [comment]
one is of a virgin aka you haha!

**Comment by newfren** [comment]
The only ingredient necessary for this not found in the dispensers is the blood, which is also not in the Booze-o-mat - if you had a syringe and a container you could mix this without bartender access, but you would require the soda dispenser so it would have to be at the main bar.

That said, I'm a fan of new drinks so 👍

**Comment by Robustin** [comment]
I wasn't able to find lemon juice outside of a booze-o-mat.

**Comment by Bawhoppen** [comment]
Just grind an actual lemon

**Comment by newfren** [comment]
https://puu.sh/z8C6J/69a5858b9f.png

Second row, third from the left.

**Comment by ExcessiveUseOfCobblestone** [comment]
Robustin was prob looking in the actual booze-o-mat and not dispensers.

**Comment by KorPhaeron** [comment]
Good cult nerf, love the excuse to just arrest the whole crew as sec once someone drinks this

**Pull request merged**